apps/src/gamelab/AnimationTab/AnimationListItem.jsx

```diff
@@ -219,7 +239,11 @@ const AnimationListItem = React.createClass({
         <div style={arrowStyle}></div>
         <ListItemThumbnail
           ref="thumbnail"
-          animationProps={Object.assign({}, this.props.animationProps, {frameDelay: this.state.frameDelay})}
+          animationProps={
+            Object.assign({},
+            this.props.animationList.propsByKey[this.props.animationKey],
+            {frameDelay: this.state.frameDelay})
+          }
```

Comment:
I wonder if we should build our composed `animationProps` at the top of the render method once, and then use it everywhere to ensure consistency (we don't really _need_ anything below this layer to see the original frameDelay value, do we?).